adding security consideration on reflection attack (#12)
tlswg/draft-ietf-tls-ticketrequest: draft-ietf-tls-ticketrequests.md

hints provided by servers, clients SHOULD dispose of pooled tickets after some reasonable
amount of time that mimics the ticket rotation period.

> In some cases, a server may sent NewSessionTicket immediately upon sending its Finished rather

Reviewer: ```suggestion
In some cases, a server may send NewSessionTicket immediately upon sending its Finished rather
```

the server Finished message rather than waiting for the client Finished. If the server

> has not verified the client's ownership of the IP address, e.g., with the TLS 

Reviewer: ```suggestion
has not verified the client's ownership of its IP address, e.g., with the TLS 
```

hints provided by servers, clients SHOULD dispose of pooled tickets after some reasonable
amount of time that mimics the ticket rotation period.

In some cases, a server may send NewSessionTicket messages immediately upon sending 
the server Finished message rather than waiting for the client Finished. If the server

> has not verified the client's ownership of the IP address - for example with the TLS 

Reviewer: ```suggestion
has not verified the client's ownership of the IP address, e.g., with the TLS 
```

> Cookie mechanism - ,  an attacker may take advantage of this behavior to create

Reviewer: ```suggestion
Cookie extension (see {{RFC8446}}; Section 4.2.2), an attacker may take advantage of this behavior to create
```

> an amplification attack proportional to the count value toward a target by performing the key

Reviewer: ```suggestion
an amplification attack proportional to the count value toward a target by performing a key
```

> exchange over UDP with spoofed packets. The limit on the number of NewSessionTicket messages

Reviewer: ```suggestion
exchange over UDP with spoofed packets. Server should limit the number of NewSessionTicket messages
```

> exchange over UDP with spoofed packets. Server should limit the number of NewSessionTicket messages

Reviewer 2: Overall I like this PR but that last sentence doesn't parse for me. What does it mean?

Reviewer: I think the suggestions got messed up -- it should be fine with the proposed edits!

Reviewer 2: I think I'm somewhat understanding it, but it still feels overly complicated. How about we replace the last sentence with `Servers SHOULD limit the number of NewSessionTicket messages they send until they have verified the client's ownership of its IP address.`

Reviewer: @mglt can we go with David's suggestion above?

> exchange over UDP with spoofed packets. Server should limit the number of NewSessionTicket messages

Reviewer: ```suggestion
exchange over UDP with spoofed packets. Servers should limit the number of NewSessionTicket messages
```

> sent in response to a "ticket_request"  MUST be based on the applicability and the amplification

Reviewer: ```suggestion
sent in response to a "ticket_request" based on the applicability and the amplification
```

> sent in response to a "ticket_request" should be based on the applicability and the amplification

Reviewer: ```suggestion
sent in response to a "ticket_request" based on the applicability and the amplification
```

> sent in response to a "ticket_request"  should be based on the applicability and the amplification

Reviewer: ```suggestion
sent in response to a "ticket_request" should be based on the applicability and the amplification
```
